cpp/include/cudf/io/orc.hpp

```diff
@@ -644,14 +661,14 @@ class orc_writer_options_builder {
   }
 
   /**
-   * @brief Enable/Disable writing column statistics.
+   * @brief Choose granularity of column statistics to be written
```

Comment:
same here

Reply:
Fixed documentation. Additionally, changed all setters for `stats_freq`.